src/main/java/org/icgc/argo/program_service/converter/Grpc2JsonConverter.java

```diff
@@ -121,6 +121,30 @@ public ProgramDetailsDTO prepareGetProgramResponse(GetProgramResponse response)
     return programDetailsDTO;
   }
 
+  public ProgramDetailsDTO prepareGetProgramResponse(ProgramDetails programDetails) {
+
+    ProgramDetailsDTO programDetailsDTO = new ProgramDetailsDTO();
+    try {
+      String responseJson = JsonFormat.printer().print(programDetails);
+      objectMapper =
+          JsonMapper.builder()
+              .disable(SerializationFeature.FAIL_ON_EMPTY_BEANS)
+              .enable(MapperFeature.ACCEPT_CASE_INSENSITIVE_ENUMS)
+              .configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false)
+              .build();
+
+      programDetailsDTO = objectMapper.readValue(responseJson, ProgramDetailsDTO.class);
+
+    } catch (JsonMappingException e) {
+      e.printStackTrace();
+    } catch (JsonProcessingException e) {
+      e.printStackTrace();
+    } catch (InvalidProtocolBufferException e) {
+      e.printStackTrace();
+    }

```

Comment:
I wrote a comment here initially asking why we are suppressing so many errors... but of course this is in grpc code that we are deprecating. Can you add a code comment calling this out to explain why we are happy to suppress errors here?

Reply:
Handled all exceptions as ProgramRuntimeException, throwing to the caller with the error message